src/main/kotlin/org/rust/ide/injected/RsStringLiteralEscaper.kt

```diff
@@ -52,7 +53,7 @@ private class RsRawStringLiteralEscaper(host: RsLitExpr) : LiteralTextEscaper<Rs
 }
 
 fun escaperForLiteral(lit: RsLitExpr): LiteralTextEscaper<RsLitExpr> {
-    val elementType = lit.firstChild?.elementType
+    val elementType = lit.node.findChildByType(RS_LITERALS)?.elementType
```

Comment:
Hm, looks like I didn't answer it in the corresponding review.
How do we ensure that we call `escaperForLiteral` only for string literals?